contentreview-impl/impl/src/java/org/sakaiproject/contentreview/impl/turnitin/TurnitinReviewServiceImpl.java

```diff
@@ -813,7 +821,7 @@ public void syncGrades(Map<String,Object>data){
                                     try{
                                         grade = ((CharacterData) (((Element)(objects.item(i))).getElementsByTagName("score").item(0).getFirstChild())).getData().trim();
                                         reportTable.put("grade"+tiiUserId, Integer.valueOf(grade));
-                                    } catch(Exception e){
+                                    } catch(DOMException | NumberFormatException e){
```

Comment:
I think the old catch would have also hidden a NullPointerException if the XML wasn't as expected (eg no score tag). Should we also be catching these here?